### DISCUSSION TITLE
Display pill group -- Variant linked not working on Figma

### DISCUSSION THREAD

**Discussion by andoyle** [twilio-labs/paste#3427] [Help: Figma] [answered]
![Screenshot 2023-08-18 at 14 09 15](https://github.com/twilio-labs/paste/assets/69856100/0e23ef80-0f9e-4656-b90b-1fd5e433efe7)

**Comment by loreina** [comment]
Hi @andoyle,

I've got a fix for this in my branch, which should roll out in the next day or two. Will follow up here once that's published. Thanks for reporting this!

**Comment by loreina** [comment, marked as answer]
Hi @andoyle,

The fix has been published in the Paste Components library! Thanks for waiting!
